tests/generators/ssz_generic/ssz_container.py

```diff
@@ -105,7 +105,7 @@ def invalid_cases():
                          RandomizationMode.mode_one_count,
                          RandomizationMode.mode_max_count]:
                 if len(offsets) != 0:
```

Comment:
The two `if len(offsets) != 0` in code blocks look like some wrong logic:
```python
        if len(offsets) != 0:
            # Note: there are many more ways to have invalid offsets,
            # these are just example to get clients started looking into hardening ssz.
            for mode in [RandomizationMode.mode_random,
                         RandomizationMode.mode_nil_count,
                         RandomizationMode.mode_one_count,
                         RandomizationMode.mode_max_count]:
                if len(offsets) != 0:
                   ...
``` 

We may have to rewrite the `RandomizationMode` usages.

Reply:
Ahh, just found it was addressed #2791 in a minute.